jenkins/css-pipeline

```diff
@@ -1,4 +1,6 @@
 #!groovy​
+def server = Artifactory.server "${env.artifacoryServerID}"
+def buildInfo = Artifactory.newBuildInfo()
```

Comment:
Perhaps this is a good idea to add some meta-data regarding the build as well? Build number? Build URL?

Reply:
I've used https://jenkins02.esss.lu.se/view/CS-Studio/job/ESS-CS-Studio-Development/444/consoleFull for inspiration on which variables to include. Please search the page for "export" and you should be able to find the variables available, when comparing them to the patterns below.